src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/RedHatSeriesMenuCard/RedHatSeriesMenuCard.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useMemo, useRef } from 'react';
+import classNames from 'classnames';
+
+import RedHatInstanceTypeSeriesSizesMenuItems from '@catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/RedHatInstanceTypeSeriesMenu/RedHatInstanceTypeSeriesSizesMenuItem';
+import { instanceTypeSeriesNameMapper } from '@catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/utils/constants';
+import { RedHatInstanceTypeSeries } from '@catalog/CreateFromInstanceTypes/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/utils/types';
+import { useInstanceTypeVMStore } from '@catalog/CreateFromInstanceTypes/state/useInstanceTypeVMStore';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { readableSizeUnit } from '@kubevirt-utils/utils/units';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuToggle,
+  Popper,
+} from '@patternfly/react-core';
+import { AngleDownIcon } from '@patternfly/react-icons';
+
+import { UseInstanceTypeCardMenuSectionValues } from '../../utils/types';
+
+import './RedHatSeriesMenuCard.scss';
+
+type RedHatSeriesMenuCardProps = {
+  rhSeriesItem: RedHatInstanceTypeSeries;
+} & UseInstanceTypeCardMenuSectionValues;
+
+const RedHatSeriesMenuCard: FC<RedHatSeriesMenuCardProps> = ({
+  rhSeriesItem,
+  menuRef,
+  activeMenu,
+  onMenuToggle,
+  onMenuSelect,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const ref = useRef<HTMLDivElement>(null);
+  const {
+    instanceTypeVMState: { selectedInstanceType },
+  } = useInstanceTypeVMStore();
+
+  const { classAnnotation, seriesName, sizes } = rhSeriesItem;
+
+  // if (['highperformance', 'server'].includes(seriesName)) return null;
+
+  const { Icon, seriesLabel, seriesTitle } = instanceTypeSeriesNameMapper[seriesName];
+
+  const isMenuExpanded = useMemo(() => seriesName === activeMenu, [activeMenu, seriesName]);
+  const isSelectedMenu = useMemo(
+    () => selectedInstanceType?.startsWith(seriesName),
+    [selectedInstanceType, seriesName],
+  );
+
+  const selectedITLabel = useMemo(() => {
+    if (isEmpty(sizes)) return null;
```

Comment:
can move to line 97

Reply:
yes, but I can't understand the reason for it, can you explain please?

Reply:
the condition is not needed, removed